src/components/CmsContent.ts

```diff
@@ -2,29 +2,42 @@ import isEqual from 'lodash/isEqual';
 import { CreateElement, Component as ComponentType, VNode } from 'vue';
 import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
 
+import { pluginOptions } from '../plugins/cms';
+
 @Component({ name: 'cms-content' })
 export default class CmsContent extends Vue {
   @Prop(String) public html!: string;
   @Prop(Object) public extra!: {};
-  @Prop(Object) public renderContext!: {};
```

Comment:
slight preference for keeping this instead of `extra` so that `renderContext` means the same thing here and in the CmsZone.vue file

Reply:
A bit torn, there isn't a need for there to be separate entities in the CmsContent component, but they are different in the zone since one of them triggers a refresh of the content (extra) and the other one is used to pass in reactive properties to the component (renderContext). How about we just keep one thing at the Content level, but call it renderContext instead of extra?

I checked the app code, and it looks like we aren't using cms-content with extra anywhere outside of vue-phoenix, so it should be a safe change.